lang/en.json

```
@@ -263,61 +263,61 @@
+        "DashboardVersionsHelp2": "To get more info about a specific version, simply hover over the name to find out when the entity was created, last edited and what type it is.",
         "DashboardVersionsHelp3": "Learn more about the Dashboard in the %1$sTag Manager Dashboard Guide%2$s.",
-        "DataLayerVariableDescription": "Reads a custom value from the Data-Layer.",
-        "DataLayerVariableHelp": "Using this variable you can access any value that is stored within the dataLayer. You can also push values to the dataLayer yourself and access them this way afterwards.",
-        "DataLayerVariableName": "Data-Layer",
+        "DataLayerVariableDescription": "Reads a custom value from the dataLayer.",
+        "DataLayerVariableHelp": "Using this variable you can access any value stored within the dataLayer. You can also push values to the dataLayer yourself and access them this way afterwards.",
+        "DataLayerVariableName": "dataLayer",
         "DefaultContainer": "Default Container",
         "DefaultValue": "Default value",
-        "DefaultValueHelp": "You can configure a default value which will be used if the variable does not return a value. Please note an empty string ('') is considered to be a value and will not fall back to the default value, configure a lookup value for this case if needed. Also note that the default value will be applied before the lookup table is evaluated.",
-        "DeleteContainerConfirm": "Are you sure you want to delete this container? You cannot undo the deletion. Please also make sure to remove all embedded code snippets for this container from your website as the container files will no longer be available after you have deleted this container.",
-        "DeleteTagConfirm": "Are you sure you want to delete this tag? You cannot undo the deletion.",
-        "DeleteTriggerConfirm": "Are you sure you want to delete this trigger? You cannot undo the deletion.",
-        "DeleteVariableConfirm": "Are you sure you want to delete this variable? You cannot undo the deletion.",
-        "DeleteVersionConfirm": "Are you sure you want to delete this version? You cannot undo the deletion.",
+        "DefaultValueHelp": "Set a default value to use if the variable does not return a value. Please note an empty string ('') is consideree a value and will not fall back to the default value, configure a lookup value if needed in this case. Also note that the default value will be applied before the lookup table is evaluated.",
+        "DeleteContainerConfirm": "Delete this container for good? Please also make sure to remove all embedded code snippets for this container from your website as the container files will no longer be available after you have deleted this container.",
+        "DeleteTagConfirm": "Delete this tag for good?",
```

```suggestion
        "DeleteTagConfirm": "Delete this tag permanently?",
```
(I would go without "permanently", but I understand its importance)
Leaving this open, as there are plenty "for good" strings.